vars/Debian.yml

```diff
@@ -0,0 +1 @@
+iptables_save_command: "/etc/init.d/iptables-persistent save"
```

Comment:
Had to look up where I was using `with_first_found` again... Thanks for taking advantage of what is already there :)

Reply:
👍